boards/arm/nucleo_f091rc/nucleo_f091rc.yaml

```diff
@@ -7,3 +7,6 @@ toolchain:
   - gnuarmemb
 ram: 32
 flash: 256
+supported:
+  - i2c
+  - spi
```

Comment:
spi too? Do we then not need to add a nucleo_f091rc.conf file in tests/driver/spi/spi_loopback/boards/?

Reply:
Ah, good point. I'll remove spi from this PR.

@erwango You may want to look into this.